### PR TITLE
Update PR template with testing instruction guidelines

### PR DESCRIPTION
**Pull request by reykjalin**
# Changes proposed in this Pull Request:

Update the PR template with testing instruction guidelines to encourage more detailed testing instructions.

# Testing instructions

N/A

-------------------
- [x] Make sure your changes respect [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [x] Did you make changes, or create a **new .js file**? If **Gruntfile.js** exists in the repo, make sure to run `grunt`.

### CODE REVIEW

**Comment by asumaran** [comment]
@reykjalin I think there are more changes than intended in this PR

**Comment by reykjalin** [comment]
@asumaran yeah, sorry about that I accidentally branched from `trunk` not `develop` 🤦

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ Images or gifs: Include before and after screenshots or gifs/videos when it make
 
 Testing instructions: How should this be tested and how can a reviewer test the end-user functionality? Are there known issues that you plan to address in a future PR? Are there any side effects that readers should be aware of?
 
+(remove me) Please follow the following guidelines when writing testing instructions:
+
+- If there is no similar flow in the [Critical flows](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Critical-flows) include screenshots.
```
Comment (asumaran) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'm no native english speaker but here a "," ("Critical flows,") would improve the spelling I believe.

**Review by asumaran — APPROVED** [review]
LGTM. Only a minor nitpick, feel free to ignore.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (reykjalin) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good point, I'll change this to "Include screenshots if there is no similar flow in the Critical flows" 🙂

Comment (reykjalin) [reply .github/PULL_REQUEST_TEMPLATE.md]: Changed in c7a47eb8544c1c55dbb269cd9f1a6af2ce18eb5c,  let me know what you think!

Comment (asumaran) [reply .github/PULL_REQUEST_TEMPLATE.md]: Looks good!

**Pull request merged**